webpack/assets/javascripts/react_app/pages/common/PageLayout/PageLayout.js

```diff
@@ -10,6 +10,9 @@ const PageLayout = ({
   header,
   searchable,
   searchProps,
+  searchQuery,
+  searchLoading,
```

Comment:
why do you need to control the Autocomplete loading spinner?
the loading spinner should be visible only when an API call is being executed.

Reply:
it will only be visible when an API call is being executed, now with pagination aswell.
I want to show it instead of the layout spinner which is legacy.

Reply:
@glekner can you explain in which use case will that use the autocompleter spinner? e.g. when i click on next page, you plan to reuse the autocompleter spinner?

Reply:
I hope future views will be written like I wrote the audits, the next and previous pages are cached so there won't be any spinner shown. I want to show the spinner only on two scenarios: increasing per page and navigating to a page which is not cached.
The less spinners the better :)

Reply:
shouldnt it be done in similar fashion to how we did the model page?

Reply:
the model page still uses Turbolinks, so i'm not sure what you mean. The way I see it, the Layout spinner is for legacy navigation, the autocomplete spinner is for API.

Reply:
IMHO the autocomplete spinner isn't connected in context to page rendering..
instead, I would suggest not showing spinners at all -OR-
move the layout spinner from legacy into react, and on the legacy side through `window.tfm`.

Reply:
I tend to agree with @Laviro, it doesn't make sense to me that pagination click show the autocomplete spinner. imo the layout spinner makes more sense here.